readme.md

```diff
@@ -456,6 +458,14 @@ test.cb(t => {
 });
 ```
 
+### Optional TAP output
+
+AVA can generate TAP output via `--tap` option for use with any 3rd-party reporters.
```

Comment:
`3rd-party reporters` => `TAP reporter` and linkify it to `https://github.com/sindresorhus/awesome-tap#reporters`